frontend/public/components/modals/column-management-modal.tsx

```diff
@@ -48,7 +48,9 @@ const DataListRow: React.FC<DataListRowProps> = ({
       <DataListItemCells
         dataListCells={[
           <DataListCell id={`table-column-management-item-${column.id}`} key={column.id}>
-            {column.title}
+            <label className="co-label--plain" htmlFor={column.id}>
```

Comment:
I think this is better, but I'm a little worried that it makes assumptions about the markup Patternfly generates (that the checkbox will have this `id`). 

I feel like `DataList` is the wrong component. We're using it for checkboxs, but it's really meant to be more of a table with selectable rows. That's why we're having accessibility problems.